x-pack/legacy/plugins/canvas/canvas_plugin_src/expression_types/embeddable.ts

```diff
@@ -5,11 +5,11 @@
  */
 
 import { ExpressionType } from 'src/plugins/expressions/public';
-import { EmbeddableInput } from 'src/legacy/core_plugins/embeddable_api/public/np_ready/public';
+import { EmbeddableInput } from '../../../../../../src/legacy/core_plugins/embeddable_api/public/np_ready/public';
```

Comment:
Embeddables are in the New Platform, you can import everything from there.

```suggestion
import { EmbeddableInput } from '../../../../../../src/plugins/embeddable/public';
```